openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -187,6 +194,11 @@ def validate_name(self, new_name):
             )
         return new_name
 
+    @staticmethod
+    def validate_gender(value):
+        """Returns gender's default value if value is empty string."""
+        return value or UserProfile.GENDER_OTHER
```

Comment:
I think We can use GENDER_DEFAULT property and delete other extra 'GENDER_*' properties. What do you think about it?